docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -335,7 +335,9 @@ metrics-category=["BLOCKCHAIN","PEERS","PROCESS"]
 ```
 
 Comma separated list of categories for which to track metrics. The default is all categories: 
```

Comment:
Maybe remove the word "all", since the default is not all the categories

Reply:
Done.